Algorithm/QCAlgorithm.Framework.cs

```diff
@@ -382,6 +383,22 @@ public void EmitInsights(params Insight[] insights)
                 return;
             }
 
+            // Verify insights are valid
+            var validInsights = new List<Insight>();
+            foreach(var insight in insights){
+                if (Securities[insight.Symbol].IsDelisted)
+                {
+                    Error($"QCAlgorithm.EmitInsights(): Cannot emit insight for delisted security {insight.Symbol}");
```

Comment:
Calling `Error` will add the message to the user log and consume his quota, and it can also be a repetitive annoying message. Perhaps we should find a solution that only logs once for the first time it happens just to let the user know that the `Insight` was discarded.

Reply:
I went ahead and changed the logic to post one warning, informing the user that they will be discarded